docs/guide/PINs/PIN-2-Result-Handlers.md

```diff
@@ -36,6 +36,6 @@ Additionally, note that this means that if handling of data fails for any reason
 
 Re-orchestrating our data flow in this way will gain us a few things:
 
-- the system will be more efficient, in that only those results which _need_ handling for the system to work will be handled
+- the system will be more efficient, in that only those results that _need_ handling for the system to work will be handled
```

Comment:
you're all over this very specific grammatical error 👏

Reply:
It was pointed out to me once that I did it a lot and now I can't unsee it...